src/components/KonnectorUpdateInfo/index.jsx

```diff
@@ -58,7 +58,7 @@ const KonnectorUpdateInfo = ({ outdatedKonnectors }) => {
             className="u-mh-0"
             label={t('KonnectorUpdateInfo.cta')}
             icon="openwith"
-            href={url}
+            href={`${url}&pendingUpdate=true`}
```

Comment:
Just searched in the store, I didn't find anything related to this querystring. Can you tell me more about it?

Reply:
J'ai approvué un peu vite : 
Tu as le pendingUpdate en argument ici https://github.com/cozy/cozy-banks/blob/4f06282d18f26b2ea96a258dcd0b9c3a02c29439/src/components/KonnectorUpdateInfo/index.jsx#L27 

Du coup c'est plus le hook ici https://github.com/cozy/cozy-banks/blob/master/src/hooks/useRedirectionURL.js#L8 à corriger